snippets/opm-rhel8-known-issue.adoc

```diff
@@ -0,0 +1,8 @@
+// Text snippet included in the following modules:
+//
+// * modules/olm-installing-opm.adoc
+// * release_notes/ocp-4-15-release-notes.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+There is currently a known issue where the version of the `opm` CLI tool released with {product-title} {product-version} does not support RHEL 8. As a workaround, RHEL 8 users can navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.14/[OpenShift mirror site] and download the latest version of the tarball released with {product-title} 4.14.
```

Comment:
@grokspawn Do you have a Jira ticket for the RHEL8/ART work that I can link to at the end of this snippet?

Reply:
Negative.  Will have one once we know what the container will look like.  It's already changed once.